### PR TITLE
Fix #661 the same rules when defining abbreviations

### LINKED ISSUE #661: §1: Use the same rules across the spec when defining abbreviations
|Version|Source|Clause(s)|
|:---:|:---:|:---:|
|AOM Working Group Draft, 17 July 2023|Apple|§1|

#### Comment:
For example:
<blockquote>
This specification defines an immersive audio model and formats (IAMF) to provide ...
</blockquote>

should use capital letters for every letter Immersive Audio Model and Formats (IAMF).

Another example:
<blockquote>
left top front (Ltf) and right top front (Rtf)
</blockquote>

should it rather be LTF/RTF or ltf/rtf?

-----
#### Suggestion:
Align abbreviation definitions across the specification.

### PR DESCRIPTION
**Pull request by sunghee-hwang**
***
<a href="https://pr-preview.s3.amazonaws.com/AOMediaCodec/iamf/pull/701.html" title="Last updated on Aug 23, 2023, 6:16 AM UTC (12c2b53)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/AOMediaCodec/iamf/701/7faeb6f...12c2b53.html" title="Last updated on Aug 23, 2023, 6:16 AM UTC (12c2b53)">Diff</a>

### CODE REVIEW

**Comment by podborski** [comment]
#661

File: index.bs [hunk 1]
```diff
@@ -260,7 +260,7 @@ url: https://www.iso.org/standard/77752.html#; spec: MP4-PCM; type: property;
 
 # Introduction # {#introduction}
 
-This specification defines an immersive audio model and formats (IAMF) to provide an [=Immersive Audio=] experience to end-users.
+This specification defines an Immersive Audio Model and Formats (IAMF) to provide an [=Immersive Audio=] experience to end-users.
```
Comment (felicialim) [inline index.bs:263]: (Not blocking this PR) I'm not sure but should we change this from "an Immersive Audio Model and Formats" to "the Immersive Audio Model and Formats"?

**Review by felicialim — APPROVED** [review]

File: index.bs [hunk 2]
```diff
@@ -8,7 +8,7 @@ Repository: AOMediaCodec/iamf
 Shortname: iamf
 URL: https://aomediacodec.github.io/iamf/
 Date: 2023-07-17
-Abstract: This document specifies an immersive audio (IA) model, a standalone IA sequence format and an [[!ISOBMFF]]-based IA container format.
+Abstract: This document specifies an Immersive Audio (IA) model, a standalone IA sequence format, and an [[!ISOBMFF]]-based IA container format.
```
Comment (sunghee-hwang) [inline index.bs:11]:
```suggestion
Abstract: This document specifies the Immersive Audio (IA) model, the standalone IA Sequence format, and the [[!ISOBMFF]]-based IA container format.
```

File: index.bs (hunk 1 above)
Comment (sunghee-hwang) [inline index.bs:263]:
```suggestion
This specification defines the Immersive Audio Model and Formats (IAMF) to provide an [=Immersive Audio=] experience to end-users.
```

Comment (sunghee-hwang) [reply index.bs:263]: If we refer AV1 specs, then it should be the.

**Pull request merged**